lib/graphql/object_type.rb

```diff
@@ -46,9 +46,11 @@ def interfaces=(new_interfaces)
     def interfaces
       @clean_interfaces ||= begin
         ensure_defined
-        @dirty_interfaces.map { |i_type| GraphQL::BaseType.resolve_related_type(i_type) }
-      rescue
-        @dirty_interfaces
+        if @dirty_interfaces.respond_to?(:map)
```

Comment:
@rmosolgo I'm not :100: happy with this, but can't think of a better way. We seem to want to let it _pass_ here such that we can pick up the issue when validating the schema in `Schema::Validation`. Alternatively I was going to `raise ArgumentError unless Array` in `interfaces=` but this would mean changing this test and behaviour: https://github.com/rmosolgo/graphql-ruby/blob/c78d24bb1f2fbfa5b0e486c4504d2ecc334cb700/spec/graphql/schema/validation_spec.rb#L83-L88

Any ideas?